pkg/plugins/resources/cargopackage/condition_test.go

```diff
@@ -61,6 +61,18 @@ func TestCondition(t *testing.T) {
 			expectedResult: true,
 			expectedError:  false,
 		},
+		{
+			name: "Retrieving none existing not-crate-test from the filesystem index",
```

Comment:
```suggestion
			name: "Retrieving non existing not-crate-test from the filesystem index",
```

Reply:
I made the revert suggestion :D 
The type check action is not happy with "non existing"